### ISSUE TITLE
Import fails for v0.0.42

### ISSUE THREAD

**Issue by evanmays** [microsoft/guidance#9]
I'm on Python 3.11
```
pip3 install guidance==0.0.42
```
Then a simple script with
```
import guidance
```
Gets error
```
Traceback (most recent call last):
  File "scrubbed...", line 1, in <module>
    import guidance
  File "/opt/homebrew/lib/python3.11/site-packages/guidance/__init__.py", line 7, in <module>
    from ._program import Program
  File "/opt/homebrew/lib/python3.11/site-packages/guidance/_program.py", line 19, in <module>
    from . import library
  File "/opt/homebrew/lib/python3.11/site-packages/guidance/library/__init__.py", line 11, in <module>
    from ._geneach import geneach
  File "/opt/homebrew/lib/python3.11/site-packages/guidance/library/_geneach.py", line 112
    partial_output( block_content = parser_context
                                    ^^^^^^^^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
```

I run `pip3 install guidance==0.0.41` and import works.

**Comment by slundberg** [comment]
Thanks, pushed an update from the wrong machine today...

**Comment by slundberg** [comment]
should be fixed now with 0.0.43
